### PR TITLE
Gitlab

### PR DESCRIPTION
**Pull request by jgmize**
- [X] working [.gitlab-ci.yml](https://github.com/mozilla/nucleus/blob/gitlab/.gitlab-ci.yml)
- [x] Add Gitlab section to readme
- [ ] delete jenkins pipeline in followup PR

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    - meao
+    - gcp
+  only:
+    - master
+  variables:
+    NAMESPACE: nucleus-dev
+  script:
+    - docker/bin/build_images.sh
+    - docker/bin/push2dockerhub.sh
+    - bin/update-config.sh
```
Comment (jpetto) [inline .gitlab-ci.yml]: Is there a reason we can't have these two combined (adding an entry in `only`)? Or are they just separated for now to call out that one is testing and will be deleted later?

File: docker/bin/push2dockerhub.sh [hunk 2]
```diff
@@ -5,10 +5,6 @@ set -ex
 BIN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source $BIN_DIR/set_git_env_vars.sh
 
-DOCKER_USERNAME="${DOCKER_USERNAME:-mozjenkins}"
-
-docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
```
Comment (jpetto) [inline docker/bin/push2dockerhub.sh]: Assuming these are now set in the cluster?

File: docker/bin/build_images.sh [hunk 3]
```diff
@@ -23,13 +23,6 @@ function imageExists() {
     return $?
 }
 
-function dockerRun() {
-    env_file="$1"
-    image_tag="mozorg/bedrock_${2}:${GIT_COMMIT}"
-    cmd="$3"
-    docker run --rm --user $(id -u) -v "$PWD:/app" --env-file "docker/envfiles/${env_file}.env" "$image_tag" bash -c "$cmd"
-}
```
Comment (jpetto) [inline docker/bin/build_images.sh]: For my own edification, why do we no longer need this?

File: docker/bin/push2dockerhub.sh (hunk 2 above)
Comment (jgmize) [reply docker/bin/push2dockerhub.sh]: the credentials are stored on the runner instances and interactive login using a password from an env var is neither needed nor desired.

File: .gitlab-ci.yml (hunk 1 above)
Comment (jgmize) [reply .gitlab-ci.yml]: as discussed on zoom, let's keep both jobs for now, and we can use the gitlab branch in the furture for testing new versions of .gitlab-ci.yml outside of the master branch.

**Pull request merged**